scripts/latent_alignment_predictions.py

```diff
@@ -0,0 +1,53 @@
+
+#! /usr/bin/env python
+
+# pylint: disable=invalid-name,wrong-import-position,protected-access
+import sys
+import os
+import gzip
+import argparse
+import json
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(os.path.join(__file__, os.pardir)))))
+
+from allennlp.data.dataset_readers import LatentAlignmentDatasetReader
+from allennlp.models.archival import load_archive
+
+def make_data(input_examples_file: str,
+              archived_model_file: str) -> None:
+    reader = LatentAlignmentDatasetReader(max_logical_forms=200)
+    dataset = reader.read(input_examples_file)
+    archive = load_archive(archived_model_file)
+    model = archive.model
+    model.eval()
+
+
+    with open(input_examples_file, "r") as data_file:
+        examples = json.load(data_file)
+
+    acc = 0.0
+    total = 0.0
+    x = []
+    for example, instance in zip(examples, dataset):
+        utterance, lf = example
+        if len(lf) <= 10: acc += 1.0
```

Comment:
I'm guessing you don't need this line, and the two below it, and you need to uncomment the ones where you're actually using the model. I would also recommend giving a more informative name to this script, something like `evaluate_latent_alignment`.

Reply:
And may be move the latent alignment scripts to a directory called `scripts/latent_alignment`